app/models/password_reset.rb

```diff
@@ -0,0 +1,36 @@
+class PasswordReset < ActiveRecord::Base
```

Comment:
No. Clearance models are internal to clearance and not generate. User is a template because we expect you already have a user class of some sort.